Detections/AzureDevOps/AzDOPatSessionMisuse.yaml

```diff
@@ -0,0 +1,44 @@
+id: ac891683-53c3-4f86-86b4-c361708e2b2b
+name: Azure DevOps Personal Access Token (PAT) misuse
+description: |
+  'Description: This Alert detects whenever a PAT is used in ways that PATs are not normally used.  May require Allowlisting and baselining.
+  Use this query for baselining:
+  AzureDevOpsAuditing
+  | distinct OperationName'
+severity: High
+queryFrequency: 1h
+queryPeriod: 3h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+  - Impact
+query: |
+
+  // Allowlisted UPNs should likely stay empty
```

Comment:
Add in timeframe parameter - let timeframe = ago(1d);

Reply:
fixed in next commit